### ISSUE TITLE
Reproduce IAP training results

### ISSUE THREAD

**Issue by fkxie** [materialsvirtuallab/m3gnet#61]
Hi,
I'm trying to reproduce the `energy`, `force` and `stress` MAE results reported in the article (0.035 eV/atom, 0.072eV/Å and 0.41GPa), but can not reach those results for now. 

Here are my confusions:

> The Adam optimizer was used with initial learning rate of 0.001, with a cosine decay to
1% of the original value in 100 epochs. During the optimization, the validation metric values
were used to monitor the model convergence, and training was stopped if the validation
metric did not improve for 200 epochs. 

1. The learning rate decay from 1e-3 to 1e-5 in 100 epochs. How large is the lr when epochs greater than 100, will it still decay or keep 1e-5 ?

2. How many epochs would the model converge approximately ?

I would appreciate that if you can provide more training details,

Thanks in advance!
